bin/init.js

```diff
@@ -81,6 +98,81 @@ rl.on( 'close', () => {
 	process.exit( 0 );
 } );
 
+/**
+ * Run husky setup
+ *
+ * @return {void}
+ */
+const runHuskySetup = () => {
+	console.log( info.success( '\nInstalling husky...' ) );
+
+	// Check if .git file exists.
+	const gitDir = path.resolve( getRoot(), '.git' );
+	if ( ! fs.existsSync( gitDir ) ) {
```

Comment:
`husky` setup would fail where VCS is set up in ancestor directories. Or Do we have a way to set it up by looking up the tree, something like how other config files work, i.e., eslintrc, gitignore etc. 🤔 

cc: @thelovekesh